spec/serializers/rest/admin/cohort_serializer_spec.rb

```diff
@@ -14,6 +14,8 @@
           'data' => be_a(Array),
           'period' => /2024-01-01/
         )
+        expect { DateTime.rfc3339(subject['period']) }.to_not raise_error
+        subject['data'].each { |datum| expect {DateTime.rfc3339(datum['date'])}.to_not raise_error}
```

Comment:
This is not a code review item per se, but I'd like to offer you a virtual handshake for your use of `datum` here. I hope it was truly enjoyable to do that.